### ISSUE TITLE
Kills Gnome 3.32 (After upgrade from Ubuntu 18.10 to 19.04)

### ISSUE THREAD

**Issue by marcinmilewski1** [petres/gnome-shell-extension-extensions#17]
Extension causes a login loop after upgrafe from Ubuntu 18.10 to Ubuntu 19.04. Please see the stacktrace:

```
Apr 19 16:19:00 marcin-Precision-5510 gnome-shell[5354]: Some code accessed the property 'ScrollablePopupMenu' on the module 'scrollablePopupMenu'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 19 16:19:00 marcin-Precision-5510 gnome-shell[5354]: Some code accessed the property 'PopupExtensionItem' on the module 'popupExtensionItem'. That property was defined with 'let' or 'const' inside the module. This was previously supported, but is not correct according to the ES6 standard. Any symbols to be exported from a module must be defined with 'var'. The property access will work as previously for the time being, but please fix your code anyway.
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: **
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: Gjs:ERROR:gi/object.cpp:1397:void ObjectInstance::associate_js_gobject(JSContext*, JS::HandleObject, GObject*): assertion failed: (!wrapper_is_rooted())
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: GNOME Shell crashed with signal 6
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: == Stack trace for context 0x559083fc12a0 ==
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #0   5590845bb9e8 i   resource:///org/gnome/shell/ui/panelMenu.js:15 (7f2dec966c10 @ 59)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #1   5590845bb930 i   resource:///org/gnome/shell/ui/panelMenu.js:98 (7f2dec966f70 @ 90)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #2   5590845bb878 i   /home/marcin/.local/share/gnome-shell/extensions/extensions@abteil.org/extension.js:35 (7f2dd0448550 @ 152)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #3   5590845bb7d0 i   resource:///org/gnome/gjs/modules/_legacy.js:82 (7f2decdb98b0 @ 71)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #4   5590845bb728 i   /home/marcin/.local/share/gnome-shell/extensions/extensions@abteil.org/extension.js:28 (7f2dd04484c0 @ 95)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #5   5590845bb680 i   resource:///org/gnome/gjs/modules/_legacy.js:82 (7f2decdb98b0 @ 71)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #6   5590845bb5e8 i   resource:///org/gnome/gjs/modules/_legacy.js:18 (7f2decdb94c0 @ 36)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #7   5590845bb530 i   resource:///org/gnome/gjs/modules/_legacy.js:114 (7f2decdb9a60 @ 184)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #8   5590845bb4a8 i   /home/marcin/.local/share/gnome-shell/extensions/extensions@abteil.org/extension.js:120 (7f2dd0448a60 @ 15)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #9   5590845bb3f8 i   resource:///org/gnome/shell/ui/extensionSystem.js:131 (7f2dec94a820 @ 504)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #10   5590845bb360 i   resource:///org/gnome/shell/ui/extensionSystem.js:177 (7f2dec94a940 @ 241)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #11   5590845bb2e0 i   resource:///org/gnome/shell/ui/extensionSystem.js:324 (7f2dec9500d0 @ 13)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #12   7ffd68b234a0 b   resource:///org/gnome/gjs/modules/signals.js:128 (7f2decdcb820 @ 386)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #13   5590845bb200 i   resource:///org/gnome/shell/misc/extensionUtils.js:251 (7f2dec950940 @ 454)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #14   5590845bb170 i   resource:///org/gnome/shell/misc/extensionUtils.js:257 (7f2dec950a60 @ 28)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #15   7ffd68b24b00 b   resource:///org/gnome/shell/misc/fileUtils.js:25 (7f2dec90a5e0 @ 318)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #16   5590845bb0e0 i   resource:///org/gnome/shell/misc/extensionUtils.js:256 (7f2dec9509d0 @ 72)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #17   5590845bb050 i   resource:///org/gnome/shell/ui/extensionSystem.js:326 (7f2dec950040 @ 201)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #18   5590845bafd0 i   resource:///org/gnome/shell/ui/extensionSystem.js:334 (7f2dec950160 @ 36)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #19   5590845baf50 i   resource:///org/gnome/shell/ui/extensionSystem.js:365 (7f2dec9503a0 @ 67)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #20   5590845baec8 i   resource:///org/gnome/shell/ui/extensionSystem.js:373 (7f2dec950430 @ 41)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #21   5590845bae38 i   resource:///org/gnome/shell/ui/main.js:218 (7f2decdd2ca0 @ 1039)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #22   5590845badb0 i   resource:///org/gnome/shell/ui/main.js:127 (7f2decdd2b80 @ 201)
Apr 19 16:19:00 marcin-Precision-5510 org.gnome.Shell.desktop[5354]: #23   5590845bad38 i   <main>:1 (7f2decd851f0 @ 48)
Apr 19 16:19:00 marcin-Precision-5510 gnome-session[5037]: gnome-session-binary[5037]: WARNING: Application 'org.gnome.Shell.desktop' killed by signal 6
```

I had to disable it by:
`gnome-shell-extension-tool -d extensions@abteil.org`

**Comment by petres** [comment]
which version of the extension are you using? 
i am using gnome 3.32 and it is working after the 5918bb392d03a8b8789d04c027762a9fa6b49dd2 commit

**Comment by marcinmilewski1** [comment]
This issue probably can be closed, since it's working in Gnome 3.32. The problem is that the old version caused the problem (login loop) during the upgrading gnome from 3.30 to 3.32.
